Library/Homebrew/diagnostic.rb

```diff
@@ -98,25 +98,6 @@ def check_path_for_trailing_slashes
         EOS
       end
 
-      # Installing MacGPG2 interferes with Homebrew in a big way
-      # https://github.com/GPGTools/MacGPG2
-      def check_for_macgpg2
```

Comment:
Nope. Last upstream stable release that caused this problem was 3 years ago, last upstream unstable release that caused this problem was 4 years ago.

At this point I think it's safe to treat as an obscure user configuration issue rather than something Homebrew needs to be constantly vigilant for. Even the newest GPGTools/MacGPG2 is supported back to 10.6, so there's no reason for anyone to be running a version from 3+ years ago.